homework/Classes_scope_ and_namespaces/task3.py

```diff
@@ -0,0 +1,63 @@
+
+
+CHANNELS = ["BBC", "Discovery", "TV1000"]
```

Comment:
I'd suggest using a tuple instead of a list here in order to make sure you won't modify it unintentionally in any part of code